rbkit-lib/ui/appmainwindow.cpp

```diff
@@ -8,8 +8,36 @@
 
 AppMainwindow::AppMainwindow(QWidget *parent) : QMainWindow(parent)
 {
-    centralWidget = new CentralWidget(this);
-    setCentralWidget(centralWidget);
+    QWidget *mainWidget = new QWidget(this);
+
+    mainLayout = new QVBoxLayout();
```

Comment:
shared pointers! these `new`s are never deleted. im not sure if Qt automatically handles deletes.